src/test/ui/associated-type-bounds/ambiguous-associated-type.rs

```diff
@@ -0,0 +1,12 @@
+// build-pass (FIXME(62277): could be check-pass?)
```

Comment:
```suggestion
// check-pass
```

This is just about type checking :)